src/Appium.Net/Appium/Android/AndroidDriver.cs

```diff
@@ -211,6 +211,27 @@ public void SetGsmSignalStrength(GsmSignalStrength gsmSignalStrength)
 
         #endregion
 
+        #region Device System
+
+        /// <summary>
+        /// Open the notifications 
+        /// </summary>
+        public void OpenNotifications() => AndroidCommandExecutionHelper.OpenNotifications(this);
+
+        /// <summary>
+        /// Retrieve visibility and bounds information of the status and navigation bars
+        /// </summary>
+        /// <returns>A dictionary whose keys are <code>statusBar</code><code>navigationBar</code></returns>
```

Comment:
could we also mention the expected types of dict values?

Reply:
Sure. change has been made.